### DISCUSSION TITLE
Building for UE4Editor 4.26

### DISCUSSION THREAD

**Discussion by brybalicious** [microdee/OpenUnrealSpaceMouse#22] [Q&A]
Somehow I am unable to build for 4.26. I downloaded release 1.0.1 for 4.26 and deleted the Intermediate and Binaries folders, and placed in a folder /myproject/Plugins/

I get the prompts asking to build, but builds are unsuccessful. Any ideas?

Thanks

**Comment by microdee** [comment]
can you try building it from Visual Studio?

**Comment by jpaulLoginNotAvailable** [comment]
if someone manages to make this work for 4.26.0 please post. cause i can't make this work neither, and this is driving me mad

**Reply by microdee** [reply]
Please try this new release https://github.com/microdee/UE4-SpaceMouse/releases/tag/1.1.0

**Comment by microdee** [comment]
well it compiles for me fine for 4.26 in Visual Studio and works as intended so far. Anyway a major 1.1 update is on its way

**Comment by microdee** [comment]
I also discovered that why the 4.26 package is faulty, now for some god forsaken reason from 4.26 apparently you have to actively only whitelist windows if you don't want your plugin to be built for Linux as well. Because I didn't have Linux toolchain installed and the UBT now instead of skipping building for Linux and carrying on it just fails instead. Wonderful. Now I have a choice of whitelisting only Windows platform or fix this issue and the Linux specific build errors I get in HIDUE.

**Comment by jpaulLoginNotAvailable** [comment]
yeah but the thing is you have programming knowledge, and compiling is like breathing for you. for me that means at least half a day, or a day of search. Anyways many, many thanks for the update

**Reply by vanengelandt** [reply]
Just installed it under 4.26 works very well thank you for your development sheers
